### ISSUE TITLE
UrlFormat: Invalid URL

### ISSUE THREAD

**Issue by ethikz** [brigade/scss-lint#890]
I think there should be a way to whitelist certain things.  I see that data has been whitelisted but we use variables in our css sometimes because it is put into a cms.

```
UrlFormat: Invalid URL `/response/${siteId}/artifact/${campaignAndToolPath}/assets/fonts/roboto-light-webfont.woff2`: bad URI(is not URI?): /response/${siteId}/artifact/${campaignAndToolPath}/assets/fonts/roboto-light-webfont.woff2
```

Maybe having a way to do

```
UrlFormat:
    enabled: true
    ignore or exclude: [
       - '${everythingbetween}'
       - '{{everything between}}'
    ]
```

**Comment by trotzig** [comment]
> we use variables in our css sometimes because it is put into a cms

Can you explain this a little more? Is the `${siteId}` variable part meant to be in the serialized css code, and your cms handles replacing it (in a post-processing step of some kind)? In that case, I think it would be more pragmatic to just turn off the `UrlFormat` check.

**Comment by ethikz** [comment]
Yes you are correct but I still think that there should be some exclusions because not all CSS files will have this in which case the `UrlFormat` would still be needed.

**Comment by trotzig** [comment]
Can you provide a small repro case for this behavior? I'm not able to reproduce it: 

```bash
⨠ echo "
.foo {
  background: url('/foo/${someVar}/bar.png');
}
" | scss-lint --stdin-file-path foo.scss
```

yields no errors.

Also, it would be helpful if you could report back what version of scss-lint you are using.

**Comment by ethikz** [comment]
When running `scss-lint -v` it returns `scss-lint 0.52.0`

`.scss-lint.yml`

```
UrlFormat:
    enabled: true
```

`src/leadfusion/r-client-skin.scss`

```
/src/leadfusion/r-client-skin.scss:12:8 [W] UrlFormat: Invalid URL `/response/${siteId}/artifact/${campaignAndToolPath}/assets/fonts/roboto-light-webfont.woff2`: bad URI(is not URI?): /response/${siteId}/artifact/${campaignAndToolPath}/assets/fonts/roboto-light-webfont.woff2
```

**Comment by trotzig** [comment]
Thanks! It turns out that my repro case had a bug. This will correctly reproduce the issue: 

```sh
⨠ echo "
.foo {
  background: url('/foo/\${someVar}/bar.woff');
}
" | scss-lint --stdin-file-path foo.scss
```

I don't know if it's worth the added complexity of being able to exclude certain instances of urls being used in your code. Unless you're able to target specific subsets of your tree using the `exclude` configuration, I think you're better off disabling the `UrlFormat` check. 

I could be convinced otherwise if you are willing to put together a PR.

**Comment by ethikz** [comment]
I'm sorry, I'm afraid I'm not that versed yet in order to submit a PR for it but maybe someone else would be willing to do so.  Until then I will just have to disable `UrlFormat`

Thanks for the help
